### PR TITLE
CephFS multiMDS Test Gaps - Functional workflow 3, 4

### PR DESCRIPTION
**Pull request by hkadam134**
# Description
Including 2 more workflows for functional test of multi-mds test gaps : 
3. Verify system behavior when multiple MDS failures occur simultaneously or in quick succession. : 
- Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-PX4WO5/

4. Reduce max_mds of filesystem and ensure standby-replay node is also reduced.
- Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-NAJN01/

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by julpark-rh** [comment]
LGTM

**Comment by hkadam134** [comment]
Added all Scenarios and a full run is executed : 

- http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-M89S0R/

File: tests/cephfs/cephfs_multi_mds/test_cephfs_multimds_functional.py [hunk 1]
```diff
@@ -291,7 +309,287 @@ def run(ceph_cluster, **kw):
...
+            "\n---------------***************---------------"
+            "\n          Scenario 2 Completed Successfully          "
+            "\n---------------***************---------------"
+            "\n"
+        )
+
+        log.info(
+            "\n"
+            "\n---------------***************---------------"
+            "\n          Scenario 3 : Verify system behavior when multiple MDS failures occur "
```
[the remark this answers is not in the document]
Comment (hkadam134) [reply tests/cephfs/cephfs_multi_mds/test_cephfs_multimds_functional.py:323]: Entire Test Run log is mentioned here - https://github.com/red-hat-storage/cephci/pull/3964#issuecomment-2277760295

**Comment by neha-gangadhar** [comment]
Please raise an MR to include the execution in pipeline for quincy, reef and squid.

**Review by neha-gangadhar — APPROVED** [review]
Approved

**Pull request merged**